official/vision/beta/projects/yolo/modeling/building_blocks/_DarkConv.py

```diff
@@ -0,0 +1,174 @@
+"""Contains common building blocks for yolo neural networks."""
+from functools import partial
+
+import tensorflow as tf
+import tensorflow.keras as ks
+import tensorflow.keras.backend as K
+from ._Identity import Identity
+
+from official.vision.beta.projects.yolo.modeling.functions.mish_activation import mish
+
+
+@ks.utils.register_keras_serializable(package='yolo')
+class DarkConv(ks.layers.Layer):
+
+  def __init__(
+      self,
+      filters=1,
+      kernel_size=(1, 1),
+      strides=(1, 1),
+      padding='same',
+      dilation_rate=(1, 1),
+      use_bias=True,
+      kernel_initializer='glorot_uniform',
+      bias_initializer='zeros',
+      bias_regularizer=None,
+      weight_decay=None,  # default find where is it is stated
+      use_bn=True,
+      use_sync_bn=False,
+      norm_momentum=0.99,
+      norm_epsilon=0.001,
+      activation='leaky',
+      leaky_alpha=0.1,
+      **kwargs):
+    '''
+        Modified Convolution layer to match that of the DarkNet Library
```

Comment:
Can you clarify a little more how this is different?

Reply:
I am working on this I did not forget.